packages/react-instantsearch-hooks/src/useConnector.ts

```diff
@@ -104,9 +115,9 @@ export function useConnector<
     return {};
   });
 
-  // We use a layout effect to add the widget to the index before the index
-  // renders, otherwise it triggers 2 network requests.
-  useLayoutEffect(() => {
+  // Using a layout effect adds the widget at the same time as rendering, which
+  // triggers a single network request, instead of two with as regular effect.
+  useIsomorphicLayoutEffect(() => {
```

Comment:
`useLayoutEffect`s have been replaced by an internal `useIsomorphicLayoutEffect` to hide the warning triggered by React on the server